src/Arcus.WebApi.Logging/RequestTrackingMiddleware.cs

```diff
@@ -210,7 +211,26 @@ private void LogRequest(string requestBody, string responseBody, HttpContext htt
             Dictionary<string, object> telemetryContext = CreateTelemetryContext(requestBody, responseBody, httpContext.Request.Headers, _logger);
             telemetryContext.Add("Body", "Request body is now available in 'RequestBody' dimension");
 
-            _logger.LogRequest(httpContext.Request, httpContext.Response, duration, telemetryContext);
+            var operationName = BuildOperationName(httpContext);
+
+            _logger.LogRequest(httpContext.Request, httpContext.Response, operationName, duration, telemetryContext);
```

Comment:
Should we make sure that we can still revert back to the original? (`operationName: null`) Because there may be scenario's where you do not want the route template but the exact route path?

Reply:
Frankly, I personally do not see such scenario's as usable.
With API's that are very often used, you'll see an explosion of requests, simply because you log the route-parameter as part of the operation-name.

Reply:
Ok,  👍